packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -145,12 +145,39 @@ - (void)showCamera {
     [[self viewControllerWithWindow:nil] presentViewController:_imagePickerController
                                                       animated:YES
                                                     completion:nil];
+  } else if (@available(iOS 9, *)){
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Error", nil) message:NSLocalizedString(@"Camera not available.", nil) preferredStyle:UIAlertControllerStyleAlert];
```

Comment:
I think it was a bad design initially. The plugin should not forcefully show a system UI with determined texts. Instead of showing an alert, I think we should send an `FlutterError` through method channel saying `camera not available`, so the developers can use this information accordingly.
While you are here, do you want to try that?